posthog/models/team/team.py

```diff
@@ -212,12 +212,9 @@ def actor_on_events_querying_enabled(self) -> bool:
             return posthoganalytics.feature_enabled(
                 "person-on-events-enabled",
                 str(self.uuid),
-                groups={"organization": str(self.organization.id)},
```

Comment:
This is a more tricky one: instead of doing a foreign key lookup that will result into a database read, let's use the _id object already cached by Django.